packages/core/src/components/widgets/BaseInput.js

```diff
@@ -61,19 +61,29 @@ function BaseInput(props) {
     return props.onChange(value === "" ? options.emptyValue : value);
   };
 
-  return (
+  return [
     <input
       className="form-control"
       readOnly={readonly}
       disabled={disabled}
       autoFocus={autofocus}
       value={value == null ? "" : value}
       {...inputProps}
+      list={schema.examples ? `examples_${inputProps.id}` : null}
       onChange={_onChange}
       onBlur={onBlur && (event => onBlur(inputProps.id, event.target.value))}
       onFocus={onFocus && (event => onFocus(inputProps.id, event.target.value))}
-    />
-  );
+    />,
+    schema.examples ? (
+      <datalist id={`examples_${inputProps.id}`}>
+        {schema.examples
+          .concat(schema.default ? [schema.default] : [])
```

Comment:
Can you add some tests that test:
- the rendering of a form with a schema with `examples`
- the rendering of a form with a schema with `examples` and a `default` defined
- the rendering of a form with a schema with no `examples` and a `default` defined

Reply:
Done :)